codecov.yaml

```diff
@@ -0,0 +1,13 @@
+codecov:
+  notify:
+    wait_for_ci: no
+
+coverage:
+  status:
+    project:
+      default:
+        # TODO
+        threshold: 1%
+
+comment:
+  layout: "diff, files"
```

Comment:
I think we want `condensed_` versions of ^these, we definitely don't want much verbosity from codecov.

Reply:
They have a website for checking diffs, so having a comment with statistics and a link is fine.

Reply:
@slimreaper35 looking at your https://github.com/slimreaper35/cachi2/pulls?q=is%3Apr+is%3Aclosed experiments, I don't think `diff` layout provides any value, so having just `files` is IMO sufficient.
One thing that I didn't like is what @ben-alkov mentioned:

> so having a comment with statistics and a link is fine

I don't want to go to some other web page to have a look at which lines are not covered. It would be awesome if there were a layout switch which could render whatever the URL takes you to and have all the necessary information here in our space.